matRad_calcDoseInit.m

```diff
@@ -6,6 +6,11 @@
     calcDoseDirect = false;
 end
 
+% check whether base data fitting without machine file is in progress
+if ~isfield(pln, 'withoutMachineFile')
+    pln.withoutMachineFile = 0;
```

Comment:
as suggested above, this should be removed

Reply:
Done.